python/nav/web/seeddb/page/prefix.py

```diff
@@ -83,6 +84,8 @@ class Meta(object):
             'net_type',
         )
 
+    attrs = set_flat_form_attributes()
+
```

Comment:
Would prob be better if this was defined higher up alongside `net_type`

Reply:
I have actually taken @podliashanyk's advise from #3105, that if we don't set any relevant attributes with `set_flat_form_attributes` we can actually just directly include `_form_content.html` without the if-else block